packages/nextjs/src/index.client.ts

```diff
@@ -38,9 +38,9 @@ const defaultBrowserTracingIntegration = new BrowserTracing({
 
 function createClientIntegrations(integrations?: UserIntegrations): UserIntegrations {
   if (integrations) {
-    return addIntegration(defaultBrowserTracingIntegration, integrations, {
-      BrowserTracing: { keyPath: 'options.routingInstrumentation', value: nextRouterInstrumentation },
-    });
+    return addIntegration(defaultBrowserTracingIntegration, integrations, [
+      ['options.routingInstrumentation', nextRouterInstrumentation],
```

Comment:
Why not pass in the integration name here? I think that would simplify the logic when trying the find the integration to update.

Reply:
Yeah, I thought of that, but I was trying not to _completely_ redo everything, LOL. Happy to do it, though.

Reply:
Something like this (like the logic I had above), so we know exactly what integration to update. 
```suggestion
      ['BrowserTracing', 'options.routingInstrumentation', nextRouterInstrumentation],
```

then we can just
```js
for (const integration of allIntegrations) {
  if integration.name === options[0] {
    update(integration, options[1], options[2]);
  }
}
```